windows_prepare_python.bat

```diff
@@ -0,0 +1,58 @@
+::
+:: Copyright (c) 2024 Intel Corporation
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http:::www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+::
+setlocal EnableExtensions EnableDelayedExpansion
+@echo off
+
+set "embeddable_python_url=https://www.python.org/ftp/python/3.9.13/python-3.9.13-embed-amd64.zip"
+
+:: Download and unpack everything 
+if exist C:\opt\python-3.9.13-embed-amd64 (
```

Comment:
Implement if %expunge% EQU 1 () parameter to delete all artifacts for the script and install a new package.

Reply:
I didn't want to install new package if installation already exists. Prepared catalog is meant to be read-only and any modifications would be made on the copies. This way we don't need additional bandwidth and time to download and unpack with every build.